pdf-converter/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "pdf-image-conveter",
+  "version": "1.0.0",
+  "description": "pdf to image converter",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "image",
+    "pdf",
+    "conveter"
+  ],
+  "author": "Parvez Shaikh",
+  "license": "ISC",
```

Comment:
What is this license?

Reply:
It is the default license when you create a Node application. Please go through this https://en.wikipedia.org/wiki/ISC_license. If you feel we don't need, let me know.